README.md

```diff
@@ -90,9 +90,8 @@ Example configuration file:
 
 ```
 # Absolute paths to the folders where generated clips will be placed.
-# `~` or `$HOME` are not supported due to mpv limitations.
```

Comment:
`~` now works thanks to the expand_path function, but  `$HOME` is still not supported.

Reply:
Ahh yes, it only works with `~` and `~~` ig.